deploy/sre-prometheus/100-telemeter-client.PrometheusRule.yaml

```
@@ -11,8 +11,10 @@ spec:
   - name: sre-telemeter-client
     rules:
     - alert: MetricsClientSendFailingSRE
-      expr: absent(rate(metricsclient_request_send{status_code="200"}[5m]) > 0) or rate(metricsclient_request_send{status_code!="200"}[5m]) > 0
-      for: 15m
+      expr: 100 * sum by (job) (rate(metricsclient_request_send{status_code="5xx"}[5m])) / sum by (job) (rate(metricsclient_request_send[5m])) > 14.4 * 0.1 and 100 * sum by (job) (rate(metricsclient_request_send{status_code="5xx"}[1h])) / sum by (job) (rate(metricsclient_request_send[1h])) > 14.4 * 0.1
```

Maybe make it a multi-line string:
```suggestion
    100 * 
        sum by (job) (rate(metricsclient_request_send{status_code="5xx"}[5m]))
    / 
        sum by (job) (rate(metricsclient_request_send[5m])) 
    > 14.4 * 0.1 
and
    100 * 
        sum by (job) (rate(metricsclient_request_send{status_code="5xx"}[1h]))
    /
        sum by (job) (rate(metricsclient_request_send[1h])) 
    > 14.4 * 0.1
```

This would roughly be the official style guidelines.

---

To make it multi-line YAML start the first line with `expr: |-`